storybookjs/storybook
# Update react-native symlink resolving and add support for flow

Issue:

The symlink resolving for react-native does not work all the time. react-native tools moved over to a new way of resolving symlinks last year: https://github.com/facebook/react-native/commit/39f83c4a12b8350e331c3a6c575a2cab64950d7a#diff-d939fe299a50129184bf6b4208f6d497

## What I did
I moved over to the new symlink method according to the react-native PR

## How to test
Run storybook on a project depending on external projects in a workspace monorepo. Make sure roots are all there (just like for react-native start) and storybook is the first root.

Is this testable with Jest or Chromatic screenshots?
No

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
